.github/actions/bench/action.yml

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Bench MLKEM-C-AArch64
+description: Run benchmarking script
+
+inputs:
+  name:
+    description: Name for the benchmarking run
+    required: true
+  perf:
+    description: Method of obtaining PMU metrics
+    required: true
+    default: 'PERF'
+    type: choice
+    options:
+      - NO
+      - PERF
+      - PMU
+      - M1
+  cflags:
+    description: CFLAGS to pass to compilation
+    required: true
+  archflags:
+    description: ARCHFLAGS to pass to compilation
+    required: true
```

Comment:
maybe we don't need the `required: true` ?

Reply:
Ok